packages/purpose-process/test/createReversePurpose.test.ts

```diff
@@ -146,6 +146,8 @@ describe("createReversePurpose", () => {
     };
 
     expect(writtenPayload.purpose).toEqual(toPurposeV2(expectedPurpose));
+    expect(writtenPayload.purpose).toEqual(toPurposeV2(purpose));
+    // TODO: should we check also isRiskAnalysisValid??
```

Comment:
Is it returned by the function? If yes, we could check it

Reply:
I agree